src/helpers.js

```diff
@@ -17,10 +18,14 @@ export const assign = Object.assign || function (target, source) {
   return target
 }
 
+export function splitEnv(contents: string) {
+  return contents.split(contents.includes('\0') ? '\0' : '\n')
```

Comment:
The entire problem is that the previous code was splitting on `\n`, What reason could there be for keeping that broken behavior?